MAUI/DataGrid/localization.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+title: Localization in .NET MAUI DataGrid Control | Syncfusion
+description: Learn here all about localization support in Syncfusion .NET MAUI DataGrid (SfDataGrid) control and more.
+platform: MAUI
+control: SfDataGrid
+documentation: ug
+---
+
+# Localization in .NET MAUI DataGrid (SfDataGrid)
+
+Localization is the process of translating the application resources into different language for the specific cultures. The `SfDataGrid` can be localized by adding `resource` file. In `SfDataGrid`, provides the support to localize the below strings
+
+   * `Load More`
+
+## Setting CurrentUICulture to the application
+
+Application culture can be changed by setting `CurrentUICulture.` in `App.xaml.cs` file.
+
+{% tabs %}
+{% highlight c# tabtitle="App.xaml.cs" hl_lines="1 3 11 15" %}
+
+using Syncfusion.Maui.DataGrid;
+using System.Globalization;
+using System.Resources;
+
+namespace MauiDataGridDemo;
+public partial class App : Application
+{
+	public App()
+	{
+		InitializeComponent();
+		CultureInfo.CurrentUICulture = new CultureInfo("fr-FR");
+      //// ResXPath => Full path of the resx file; For example : //SfDataGridResources.ResourceManager = new ResourceManager
+      // ("MauiDataGridDemo.Resources.SfDataGrid", Application.Current.GetType().Assembly);
+
+		SfDataGridResources.ResourceManager = new ResourceManager("ResxPath", Application.Current.GetType().Assembly);
+	   MainPage = new MainPage();
+	}
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Unbound Rows" src="Images\localization\maui-datagrid-localization.png" width="404"/>
```

Comment:
Alt text should be `Load more in MAUI SfDataGrid`

Reply:
modified